docs/06-contributors/999-rfcs/000-stories/story-15.md

```diff
@@ -56,21 +57,21 @@ A good example on how this may look like is Redis embedded prompt inside their d
 bring cloud;
 bring redis;
 
-extern "./tasklist_helper.js" {
-  inflight get_data: (url: str) => Json;
-  inflight create_regex: (s: str) => IMyRegExp;
-  inflight uuid() => str; 
-};
+interface IMyRegExp {
+  inflight test(s: str): bool
+}
+
+resource Helper {
```

Comment:
I want to note that accessing static members inflight still requires some work https://github.com/winglang/wing/issues/1669.

However, if these externs were moved inside of `TaskList` instead of this `Helper` I think it would be able to work with just the the new extern feature.

Reply:
I forgot to add the `static` keyword - adding now

@MarkMcCulloh , can you educate me why TaskList will work and Helper wont?

Reply:
We don't have a way for code attached to types to cross boundaries (between resources and/or between phases). #1669 should cover that (at least partially).

If you're calling an inflight static from within the same resource, technically the code is within the same file. This still doesn't actually work yet, but I've been working on extern already and I think I can have it work with a small hack. 
Having it work across resources would be much more work.

Reply:
so we only have **private** static method. It is not a bug, it is a feature 💵

Reply:
updated doc, can you take a look now?